src/index.js

```diff
@@ -44,6 +44,27 @@ export default class Kinto {
   }
 
   /**
+   * Creates a remote transformer constructor, the ES5 way.
+   *
+   * @return {RemoteTransformer}
+   */
+  static createRemoteTransformer(proto) {
+    if (!proto || typeof proto !== "object")
+      throw new Error("Expected prototype object.");
```

Comment:
Depends on your coding style; Gecko (and Mozilla)'s one states that for single line statements braces can be omitted. If we're planning on removing this rule, we'll need to update a bunch of files elsewhere; if that's the case, it's probably safe to spawn another issue.